src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -635,6 +658,11 @@ def load_arguments(self, _):
 
         c.argument('max_size_bytes', help='The new maximum size of the database expressed in bytes.')
 
+        c.argument('storage_account_type',
+                   arg_type=backup_storage_redundancy_param_type,
+                   options_list=['--backup-storage-redundancy', '--bsr'],
+                   help='Backup storage redundancy used to store backups')
+
```

Comment:
avoid duplicate. use CLIArgumentType to abstract common argument type and reuse it for command